### ISSUE TITLE
classifiedBy - AI Specification

### ISSUE THREAD

**Issue by ben-norton** [tdwg/camtrap-dp#225]
For observations classified by computer vision models, additional specifics are needed to enter information in the classifiedBy field. I think megadetector is an exception. It may be one of the only AI techniques that can be referred to by a single name. Most computer vision models are much more complex. At minimum, several parameters are needed to reference a computer vision model. The megadetector repo states the following regarding its purpose: 
"MegaDetector" - to detect animals, people, and vehicles in camera trap images. It does not identify animals to the species level, it just finds them."
You would need more than just megadetector to make an observation below the kingdom level.

**Comment by peterdesmet** [comment]
The `classifiedBy` isn't limited to a single name. E.g. we use it to express `Western Europe species model v1`. It could even contain a link/doi to a paper describing the model. And I think it could be fine to separate multiple contributors, e.g. `Western Europe species model v1 | Jim Casaer`.

If we agree on that, I guess we could update the definition to clarify that.

**Comment by kbubnicki** [comment]
Then I think that `classificationMethod` should follow the same logic #224

**Comment by peterdesmet** [comment]
@kbubnicki just a suggestion: would you prefer that we split `classifiedBy` into `classifiedBy` (single value) and `verifiedBy` (single value). Or is that going to be too limiting (because two values maximum) and confusing with the other "classified" terms?

**Comment by peterdesmet** [comment]
@kbubnicki and I discussed this and we decided **not allow multiple (` | ` separated) values for the classification terms**. It makes it harder to validate controlled values, formats etc. and doesn't allow comparison between classifications. I will indicate for all term name, that the value applies to the "most recent" action:

- `classificationMethod`: Method (most recently) used to classify the observation.
- `classifiedBy`: Name or unique identifier of the person or AI algorithm that (most recently) classified the observation.
- `classificationTimestamp`: Date and time of the (most recent) classification.
- `classificationConfidence`: Accuracy confidence of the (most recent) classification.
